compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -1413,6 +1420,29 @@ object Parsers {
       }
     }
 
+    private def makeKindProjectorTypeDef(name: TypeName, span: Option[Span]): TypeDef = {
```

Comment:
I think we should drop the `span` parameter and just have call-sites  that need to set a span do it themselves (`makeKindProjectorTypeDef(...).withSpan(...)`), it's not much longer and slightly clearer.

Reply:
Agreed—I've just pushed the change.